lib/local-aspects/iam-path/index.ts

```diff
@@ -0,0 +1,26 @@
+import { IAspect } from "aws-cdk-lib";
+import { IConstruct } from "constructs";
+import * as iam from "aws-cdk-lib/aws-iam";
+import { isCfnRole, isCfnUser, isCfnGroup } from "shared-utils";
+
+export class IamPathAspect implements IAspect {
+  private readonly iamPath: string;
+
+  constructor(iamPath: string) {
+    this.iamPath = iamPath;
+  }
+
+  public visit(node: IConstruct): void {
+    if (node instanceof iam.Role && isCfnRole(node.node.defaultChild)) {
+      node.node.defaultChild?.addPropertyOverride("Path", this.iamPath);
```

Comment:
Just noticed this `?`. The other ones don't have it so I'm assuming you forgot this one?